papertty.py

```diff
@@ -131,6 +131,16 @@ def img_diff(img1, img2):
     def ttydev(vcsa):
         """Return associated tty for vcsa device, ie. /dev/vcsa1 -> /dev/tty1"""
         return vcsa.replace("vcsa", "tty")
+    
+    def vcsudev(self, vcsa):
+        """Return character width and associated vcs(u) for vcsa device,
+           ie. for /dev/vcsa1, retunr (4, "/dev/vcsu1") if vcsu is available, or
+           (1, "/dev/vcs1") if not"""
+        dev = vcsa.replace("vcsa", "vcsu")
+        if isinstance(self.font, ImageFont.FreeTypeFont) and os.path.exists(dev):
```

Comment:
Another change if we want the static version:

```suggestion
        if os.path.exists(dev) and not force_vcs:
```